pandas/core/arrays/datetimes.py

```diff
@@ -1129,12 +1129,12 @@ def date(self):
         "The name of day in a week (ex: Friday)\n\n.. deprecated:: 0.23.0")
 
     dayofyear = _field_accessor('dayofyear', 'doy',
-                                "\nThe ordinal day of the year\n")
-    quarter = _field_accessor('quarter', 'q', "\nThe quarter of the date\n")
+                                "The ordinal day of the year")
+    quarter = _field_accessor('quarter', 'q', "The quarter of the date")
```

Comment:
If you don't mind adding periods at the end of these 3 docstrings, that would be great, so we fix that error too.

Also, if you can merge master, and check that the CI is green, so we can merge.